frontend/src/modals/ModalBase.svelte

```diff
@@ -25,6 +25,9 @@
           ev.preventDefault();
           attemptFocus(first);
         }
+      } else if (ev.key === "Escape") {
```

Comment:
I'd like your feedback on the placement of this code. It's within the `keydown` handler defined within `handleFocus`. I'm not sure if that makes sense or not.

Reply:
It's vaguely related to focus management, so that's fine IMHO.